spec/support/examples_group/shared_examples_for_compliance_mixin.rb

```diff
@@ -0,0 +1,30 @@
+shared_examples_for "MiqComplianceMixin" do
```

Comment:
MiqComplianceMixin isn't an actual thing, so I'm concerned this might get confusing.  Think you can just make this:

```suggestion
shared_examples_for "ComplianceMixin" do
```